src/SurveyDesign.jl

```diff
@@ -150,6 +150,10 @@ struct StratifiedSample <: AbstractSurveyDesign
             data[!, :weights] = weights
             data[!, :probs] = 1 ./ data[!, :weights]
         end
+        data[!, :sampsize] = sampsize
```

Comment:
This might not be necessary. In most calculations we are using `sampfraction` and in some (such as calculating variance of the mean) we also need `popsize`. We can keep it for now and remember to check if we need it after we implement most functionality.

Reply:
Hi iulia in the variance formula we have an fh term, so I have kept sampfraction